src/types/box.js

```diff
@@ -75,6 +75,10 @@ BoxAnnotation.defaults = {
     yAdjust: 0,
     width: undefined
   },
+  shadowBlur: 0,
+  shadowColor: undefined,
```

Comment:
Should probably be transparent, because setting undefined to ctx usually results it to keeping the previously set value, which could be anything leaked from some other plugin etc.

We should also look into having some kind of "common defaults", to avoid the repeated stuff.

Reply:
> Should probably be transparent, because setting undefined to ctx usually results it to keeping the previously set value, which could be anything leaked from some other plugin etc.

I had a doubt if to set to undefined or transparent. The MDN documentation says:

```
Shadows are only drawn if the shadowColor property is set to a non-transparent value. 
One of the shadowBlur, shadowOffsetX, or shadowOffsetY properties must be non-zero, as well. 
```

Now I set to transparent to be aligned with MDN doc. Nevertheless I used undefined because I see that in Datalabels, where it is used, the default was undefined but offsets are not configurable but only blur which is set to 0 for reset.